code/src/Like.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+export const Like = ( {_id} ) => {
+
+    const handleLike = (_id) => {
+        //POST like on specific comment
+        fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts/${_id}/like`, {
+          method: "POST",
+          headers: { "Content-Type":"application/json" },
+          body:"",
+        })
+    }
+
+  return  <button onClick={() => {handleLike(_id)}} className="like-button">
+            <span role="img" aria-label="Heart emoji"> ❤️ </span>
```

Comment:
Here you could add and if statement that if the button gets 1 or more clicks, it should change background color. Like this: style={{ background: hearts > 0 ? '#feacac': 'f2f0f0'}} (Then you also have to add the hearts prop in the component). On your web page the background color of the heart only changes temporarily (when clicking on it) But then it changes back to the original background color.